common/protob/messages-bitcoin.proto

```diff
@@ -130,7 +130,7 @@ message SignTx {
     optional uint32 version = 4 [default=1];            // transaction version
     optional uint32 lock_time = 5 [default=0];          // transaction lock_time
     optional uint32 expiry = 6;                         // only for Decred and Zcash
-    // optional bool overwintered = 7;                  // deprecated - only for Zcash
+    optional bool overwintered = 7 [deprecated=true];   // deprecated in 2.3.2, the field is not needed as it can be derived from `version`
```

Comment:
Do we want to rename deprecated fields (add `_` prefix?). That's what we did in the past (`PassphraseRequest._on_device` and `PassphraseAck._state`), but I have no strong opinion about the act itself. I just like consistency :)

Reply:
That makes sense, what do you think @matejcik?

Reply:
we can. For consistency, we should also rename `old_address` in `EthereumAddress`

Reply:
Fixed in 59dd184